doc/integrator/docker.rst

```diff
@@ -166,7 +166,7 @@ Config:
 
  * ``VISIBLE_WEB_HOST``: The web host visible by the browser e.g.: 'example.com'.
  * ``VISIBLE_ENTRY_POINT``: The web path visible by the browser e.g.: '/main/', default to '/'.
- * ``PGSCHEMA``: The postgres schema, used by MapServer.
+ * ``PGSCHEMA``: The PostgreSQL schema, used by MapServer.
```

Comment:
Why do we say "used by MapServer" here? It's hard to understand how it relates to what we say above in the database.rst